arm-runner/tasks/main.yml

```diff
@@ -47,6 +47,7 @@
     sudo ./svc.sh status
   args:
     chdir: "{{ path_to_runner_directory }}"
+  ignore_errors: yes
```

Comment:
Why's this necessary? Should errors be ignored if the service fails to start? :thinking: (I do see this is used elsewhere in the role, curious about those too)